docs/providers/aws/cli-reference/create.md

```diff
@@ -33,8 +33,9 @@ serverless create --template-url https://github.com/serverless/serverless/tree/m
 ```
 
 ## Options
-- `--template` or `-t` The name of one of the available templates. **Required if --template-url is not present**.
-- `--template-url` or `-u` The name of one of the available templates. **Required if --template is not present**.
+- `--template` or `-t` The name of one of the available templates. **Required if --template-url and --template-path are not present**.
+- `--template-url` or `-u` The name of one of the available templates. **Required if --template and --template-path are not present**.
+- `--template-path` The local path of your template. **Required if --template and --template-url are not present**.
```

Comment:
`create` command can be used by all providers.
Cloud you add this feature to other providers docs other than AWS?

Reply:
good catch, thanks!